cabal2nix/CHANGELOG.md

```diff
@@ -21,6 +27,11 @@
   package causing the input to stop working as intended. Users who relied
   on this resolution, will need to pass in `pkgs.nix` manually via overrides
   going forward.
+* LLVM related packages are no longer resolved to the version GHC uses for
+  `-fllvm` since such a match is usually not required. Instead, it use
+  to the global default version Nixpkgs uses unless overridden.
```

Comment:
```suggestion
  `-fllvm` since such a match is usually not required. Instead, it resolves
  to the global default version Nixpkgs uses unless overridden.
```